pkg/binlogfile/binlogger.go

```diff
@@ -187,6 +191,53 @@ func (b *binlogger) ReadFrom(from binlog.Pos, nums int32) ([]binlog.Entity, erro
 	return ents, err
 }
 
+// ReadAll reads all binlog in the directory.
+func (b *binlogger) ReadAll(stop chan struct{}) (result chan *binlog.Entity, errChan chan error) {
```

Comment:
comment for the behavior of `result` and `errChan`, like `errChan` will always return one entry (nil or some error)

Reply:
Comments are added, and `errChan` won't contain `nil` now.